page-components/featured-content-module.md

```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
+  <img src="{{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png" alt="Image of featured content module example at desktop size">
 </div>
+{:/nomarkdown}
 
-<div class="content-33 content-first">
-* Module is a minimum of 320px total in height with 30px top and bottom padding. It appears 60px underneath the header/intro paragraph of the page.
-* Background color of module is Gray 5 with a 1px Gray 50 stroke.
-* Module bleeds to the edge of the page, right sidebar and left nav.
-* Visual spans 5 columns and bleeds into right sidebar. The visual is sized at 640 x 360px (a 16:9 ratio) for optimal appearance at all screen sizes.
-* Text spans 3 columns and is left and top aligned.
+<div class="content-50 content-first">
+* Module spans the full width of the page’s content area and is a minimum of 220px tall. It appears 60px underneath the header/intro paragraph of the page.
+* Background color of module is Gray 5 with a 1px Gray 40 stroke.
+* Visual is 270px wide and a minimum of 220px tall. The height of the visual will vary at different screen sizes so that the full height of the image is always visible. The aspect ratio of the image changes with screen size to keep the full image height visible; the image will be cropped up to 23% from each side.
+* Text spans 5 columns and is left and top aligned. There is 30px of padding around all sides of the text.
 </div>
 
-<div class="content-33">
-* Heading: H2, appears 30px below category, 35 maximum character count (including spaces)
+<div class="content-50">
+* Heading: H3, 35 maximum character count (including spaces)
 * Body copy: 16px, appears 15px below the headline, 160 maximum character count (including spaces)
-* Call to action link: 16px, standard link colors, appears 30px below body copy, 30 maximum character count (including spaces)
-* Call to action button: Blue primary button, appears 30px below the body copy
-</div>
-
-<div class="content-33 content-last">
-* Category: 18px black minicon and H4 text, appears 30px from top
-* The following five categories can be used.
-  * <h4>{% include icons/date.svg %} Featured event</h4>
-  * <h4> {% include icons/speech-bubble.svg %} Featured blog</h4>
-  * <h4>{% include icons/play-round.svg %} Featured video</h4>
-  * <h4>{% include icons/settings.svg %} Featured tool</h4>
-  * <h4>{% include icons/newspaper.svg %} Featured news</h4>
-* If the content does not fit within one of the five categories, you may use this icon and prefix:
-  * <h4>{% include icons/favorite.svg %} Featured </h4>
+* Call to action link: 16px, standard link colors, appears 15px below body copy, 35 maximum character count (including spaces)
+* Play button (for FCMs with a video): 60px diameter, Black background at 75% opacity that changes to Pacific on hover, White icon, 2px Gray 5 border, centered in the visual
 </div>
 
 ## Style at tablet size
```

Comment:
Should we just get rid of this section? It doesn't seem to serve much purpose anymore.

Reply:
Agreed. I combined the desktop and tablet specs in eeb539d.